### PR TITLE
Fix GLCanvas Demo

### PR DESCRIPTION
**Pull request by Metallicow**
Add a bit more example to the cube and the cone. Made note to DO NOT USE glut. It is old.
Folks seem to always have hard time figuring out how to load images. Added image texturing example.
Also changed cone to meshmode.

Fixes Broken GLCanvas.py Demo once again.

### CODE REVIEW

File: demo/GLCanvas.py [hunk 1]
```diff
@@ -13,11 +16,17 @@
...
+    #### OpenGL.GLUT  Very old, DO NOT USE!
     haveOpenGL = True
 except ImportError:
     haveOpenGL = False
 
+## try:
+##     from PIL import Image
+##     havePIL = True
+## except ImportError:
+##     havePIL = False
```
Comment (RobinD42) [inline demo/GLCanvas.py:29]: remove this block

File: demo/GLCanvas.py [hunk 2]
```diff
@@ -13,11 +16,17 @@
     # The Python OpenGL package can be found at
     # http://PyOpenGL.sourceforge.net/
     from OpenGL.GL import *
-    from OpenGL.GLUT import *
+    from OpenGL.GLU import *
+    #### OpenGL.GLUT  Very old, DO NOT USE!
```
Comment (RobinD42) [inline demo/GLCanvas.py:20]: Just remove the line, no need to preach.

File: demo/GLCanvas.py [hunk 3]
```diff
@@ -62,14 +71,15 @@ def OnButton(self, evt):
             dlg = wx.MessageDialog(self,
                                    'The OpenGL package was not found.  You can get it at\n'
                                    'http://PyOpenGL.sourceforge.net/',
+                                   '/nor $ pip install PyOpenGL PyOpenGL_accelerate',
```
Comment (RobinD42) [inline demo/GLCanvas.py:74]:
```
Traceback (most recent call last):
  File "./GLCanvas.py", line 71, in OnButton
    dlg = wx.MessageDialog(self,
TypeError: MessageDialog(): argument 4 has unexpected type 'str'
```

**Review by RobinD42 — CHANGES_REQUESTED** [review]

File: demo/GLCanvas.py (hunk 3 above)
Comment (Metallicow) [reply demo/GLCanvas.py]: Opps. that one wasn't intended. Guess I had a brain fart. Fixed comma typo and formatting.
Also removed other requested lines.

**Pull request merged**